sdccc/src/main/java/com/draeger/medical/sdccc/tests/biceps/invariant/InvariantBicepsNormativeAnnexTest.java

```diff
@@ -42,6 +42,10 @@ void setUp() {
         this.messageStorage = injector.getInstance(MessageStorage.class);
     }
 
+    // NOTE: No PreCondition can ensure that there are Localized texts with a @Ref attribute. This is because
+    //       a device either provides a LocalizationService, in which case all Localized texts will have a @Ref attribute,
```

Comment:
This is not true, most texts will have a `@Ref`, but there is still, e.g., user-entered text and also the SystemErrorReport, in which the `@Ref` is explicitly forbidden.

Reply:
Adapted the note. Please check again.